src/sql/parts/modelComponents/splitviewContainer.component.ts

```diff
@@ -0,0 +1,167 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import 'vs/css!./flexContainer';
+
+import {
+	Component, Input, Inject, ChangeDetectorRef, forwardRef, ComponentFactoryResolver,
+	ViewChild, ViewChildren, ElementRef, Injector, OnDestroy, QueryList,
+} from '@angular/core';
+
+import { IComponent, IComponentDescriptor, IModelStore } from 'sql/parts/modelComponents/interfaces';
+import { FlexItemLayout, SplitViewLayout } from 'azdata';
+import { FlexItem } from './flexContainer.component';
+import { ContainerBase, ComponentBase } from 'sql/parts/modelComponents/componentBase';
+import { Event, Emitter } from 'vs/base/common/event';
+import { SplitView, Orientation, Sizing, IView } from 'vs/base/browser/ui/splitview/splitview';
+
+class SplitPane implements IView {
+	element: HTMLElement;
+	minimumSize: number;
+	maximumSize: number;
+	onDidChange: Event<number> = Event.None;
+	size: number;
+	layout(size: number): void {
+		this.size = size;
```

Comment:
Can logic to also change the height of the component be added? This will prevent the components from overlapping with each other.

Reply:
Done